plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -80,7 +83,7 @@ public void testRenameTable()
     {
         assertThatThrownBy(super::testRenameTable)
                 // TODO (https://github.com/trinodb/trino/issues/7204) support table rename in Phoenix
-                .hasMessageContaining("Syntax error. Encountered \"RENAME\"");
+                .hasMessageContaining("This connector does not support renaming tables");
```

Comment:
You should be able to remove the `testRenameTable` override altogether

Reply:
I can't, because it's defined in `AbstractTestDistributedQueries` and doesn't check for behaviors defined in `hasBehavior()`. I also can't just move it to `BaseConnectorTest`, where I added `testRenameTableAcrossSchemas`, because there's a lot of connectors not yet converted to use that new class.

Reply:
We can add a `@Deprecated supportsRename` in `AbstractTestDistributedQueries` (and implement it in BCT)
for the transition period until we get rid of ATDQ 

(no change requested now)